group_vars/rhcs.yml.sample

```diff
@@ -273,6 +273,16 @@ ceph_iscsi_config_dev: false
 
 #cephx: true
 
+# Cluster configuration
+#ceph_cluster_conf:
+#  global:
+#    public_network: "{{ public_network | default(omit) }}"
+#    cluster_network: "{{ cluster_network | default(omit) }}"
+#    osd_pool_default_crush_rule: "{{ osd_pool_default_crush_rule }}"
+#    ms_bind_ipv6: "{{ (ip_version == 'ipv6') | string }}"
+#    ms_bind_ipv4: "{{ (ip_version == 'ipv4') | string }}"
+#    osd_crush_chooseleaf_type: "{{ '0' if common_single_host_mode | default(false) else omit }}"
+
```

Comment:
rhcs.yml.sample should probably be dropped

Reply:
https://github.com/ceph/ceph-ansible/commit/1af387621dfdd59373a61cb38665a3553f4a96ed